### DISCUSSION TITLE
provide pytorch_android_lite version？

### DISCUSSION THREAD

**Discussion by theta666** [snakers4/silero-vad#166] [Q&A] [answered]
Since Pytorch team is updating pytorch_android to pytorch_android_lite, I found the newly built version can only use models  created by "model._save_for_lite_interpreter",instead of ".save", so do we have such version model file?

**Comment by snakers4** [comment, marked as answer]
Are you referring to this tutorial - https://pytorch.org/tutorials/prototype/lite_interpreter.html or this - https://pytorch.org/tutorials/recipes/mobile_interpreter.html?

@adamnsandle  can you try exporting using this method for a test?

**Reply by theta666** [reply]
yes, another model of my program is saved in this "lite" way, and loaded by manually built pytorch-android.aar(inside jni/arm64-v8: alibc++_shared.so ibfbjni.so  libpytorch_jni__**_lite_**.so), then I found my vad codes(using model_micro_mobile.jit) not working

**Reply by theta666** [reply]
I happened to find that I can convert the pytorch_android to pytorch_android_lite by this:
```
model = torch.jit.load('./old.jit', map_location=torch.device('cpu'))
model._save_for_lite_interpreter('lite.jit')
```
and load it on android
```
import org.pytorch.LiteModuleLoader;
Module vadmodule = LiteModuleLoader.load(modulePath);
```
results are slightly different, but okay to use
inference time is longer, 2.7ms(pytorch_android 1.8) to 3.3ms(pytorch_android_lite 1.10?) per 4000/16000 second

**Reply by snakers4** [reply]
> per 4000/16000 second

I believe that the recommended chunk sizes for 16 kHz are ` [512, 1024, 1536] ` samples

https://github.com/snakers4/silero-vad/blob/master/utils_vad.py#L196

**Reply by theta666** [reply]
thank you, I realize the result is computed using first version model

**Reply by snakers4** [reply]
please use the newest models
